tests/nnapi/nnapi_gtest.skip.aarch64-linux.acl_cl

```diff
@@ -70,6 +70,7 @@ GeneratedTests.mul_dynamic_nnfw
 GeneratedTests.neg
 GeneratedTests.neg_dynamic_nnfw
 GeneratedTests.pack_ex_dynamic_nnfw
+GeneratedTests.pow_dynamic_nnfw
```

Comment:
Could you keep alphabetical order? (please check other `*.skip.*` files) thanks.